src/utils/daos_hdlr.c

```diff
@@ -43,9 +43,12 @@
 #include "daos_api.h"
 #include "daos_fs.h"
 #include "daos_uns.h"
+#include "daos_prop.h"
 
 #include "daos_hdlr.h"
 
+#define MAX_PROP_NAME_LEN	(64)
```

Comment:
daos_parse_properties() function defines a variable char name[20]. Maybe if the code is going to have separate processing for single property handling for set-prop while supporting multiple properties at container create time they could use a common constant for the maximum property name length?

Reply:
Good idea.